modules/ROOT/partials/misc/plugin-toolbar-button-id-boilerplate.adoc

```diff
@@ -1,3 +1,7 @@
+ifndef::pluginpage[]
```

Comment:
Is this needed? `plugin-menu-item-id-boilerplate,adoc` doesn't have it so just wanted to check.

Reply:
Oops, this was definitely not meant to be here. It was an earlier experiment that failed and looks like I didn't revert properly.

Reply:
It's been removed, thanks for the catch!